sjb/config/test_cases/test_pull_request_origin_extended_image_registry.yml

```diff
@@ -11,4 +11,20 @@ extensions:
       repository: "origin"
       script: |-
         OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64/extended.test hack/env make build-extended-test
-        OPENSHIFT_SKIP_BUILD='true' KUBECONFIG=/etc/origin/master/admin.kubeconfig TEST_ONLY=true JUNIT_REPORT='true' make test-extended SUITE=core FOCUS="\[Feature:Image|\[registry\]"
\ No newline at end of file
+
+        export OPENSHIFT_SKIP_BUILD=true
+        export KUBECONFIG=/etc/origin/master/admin.kubeconfig
+        export TEST_ONLY=true
+        export JUNIT_REPORT=true
+
+        source "hack/lib/init.sh"
+        source "test/extended/setup.sh"
+
+        export TEST_REPORT_FILE_NAME=registry_suite
+        export FOCUS="\[Feature:Image|\[registry\]"
+
+        os::log::info "Running registry extended tests serially"
+        exitstatus=0
+        os::test::extended::run -- -test.timeout 6h ${TEST_EXTENDED_ARGS:-} || exitstatus=$?
+        os::test::extended::merge_junit
```

Comment:
should not need merge_junit anymore.  i'm trying to get it removed here:
https://github.com/openshift/origin/pull/16068/files